paper-typeahead-input.html

```diff
@@ -543,8 +574,29 @@
       else{
         this._suggestions = matched;
       }
+    },
+    _clearSuggestings:function() {
+      this._suggestions = [];
+      var input = Polymer.dom(this.root).querySelector('paper-input');
+      input.$.input.blur();
+    },
+    _onInputFocus:function(e, detail) {
```

Comment:
I don't know why when I test locally, the this function _onInputFocus is not triggered when input is focused. And also _onInputBlur is not triggered when lose focus. Do I miss anything to get it working?  I already set the suggestOnFocus to be true. ( but this shouldn't matter though.